src/main/java/org/baeldung/Application.java

```diff
@@ -1,4 +1,4 @@
-package org.baeldung.spring;
```

Comment:
Why are we moving this class @rajadilipkolli ?
And - more importantly - how is this related to users clicking back on the captcha?